EDEngineer/Views/MainWindowViewModel.cs

```diff
@@ -108,7 +109,35 @@ private void LoadState()
                 aggregation = GetAggregation();
             }
 
-            if (aggregation == null || !aggregation.Aggregations.Any())
+            bool loadAll = true;

```

Comment:
what's this new section for?

I would prefer not to have the empty `catch` - if this throws then something is wrong and the app should crash

Reply:
After renaming ingredients for issue #602, the app crashed because the ingredient was used by a blueprint, and the ingredient was also stored in aggregation file.
I reverted this part

Reply:
I understand - so for the upcoming update we should just discard the aggregation.json file if it exists is that correct?

Reply:
Yes, it will show "ghost ingredients" because they are stored in the aggregation file.
I removed the auto-insertion of unknown ingredients in IncrementCargo (Statecargo.cs) but I don't like this way to do, it's useful to see errors or missing data.
Instead of it, I was thinking about adding a flag in the releasenote file to reset the aggregation file the first time a specific version is used (and stored in settings once done). What do you think about it?

Reply:
yes a reset flag is a good idea:
* resets all filters (new engineer/type gets to be selected)
* aggregation is discarded

aggregation might be obsolete btw - I added that back when Elite Dangerous was not outputting a snapshot of the cargo every login and all logs had to be processed to rebuild the application state - but let's address that separately if we do